Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.17
+
+WORKDIR /
+
+COPY api api
+COPY cmd cmd
+COPY pkg pkg
+COPY test test
```

Comment:
Are testing packages necessary to run the container? Can we remove their inclusion into the destination container image?